static/js/components/Dashboard.js

```diff
@@ -7,14 +7,45 @@ import { makeProfileImageUrl } from '../util/util';
 
 class Dashboard extends React.Component {
   static propTypes = {
-    profile:    React.PropTypes.object.isRequired,
-    dashboard:  React.PropTypes.object.isRequired,
-    expander: React.PropTypes.object.isRequired,
-    dispatch: React.PropTypes.func.isRequired,
+    profile:      React.PropTypes.object.isRequired,
+    dashboard:    React.PropTypes.object,
+    expander:     React.PropTypes.object,
+    dispatch:     React.PropTypes.func.isRequired,
+    onProfile:    React.PropTypes.bool.isRequired,
   };
 
+  printDashboard = () => {
+    const { dashboard, onProfile, expander, dispatch } = this.props;
+    if ( onProfile === false ) {
+      return (
+        <div>
+          <div className="card-header">
+            Your Status
+          </div>
+          <div className="card-copy">
+            <CourseList dashboard={dashboard} expander={expander} dispatch={dispatch} />
+          </div>
+        </div>
+      );
+    }
+  }
+
+  preferredName = () => {
+    const { profile: { profile } } = this.props;
+    return profile.preferred_name || SETTINGS.name;
+  }
+
+  cardText = () => {
+    const { onProfile } = this.props;
+    if ( onProfile === true ) {
+      return `Welcome ${this.preferredName()}, let's complete your enrollment to MIT Micromasters`;
```

Comment:
@pdpinch Should this be `MIT MicroMaster's` per #236? Where is the apostrophe used?